### DISCUSSION TITLE
Base table or view not found: 1146 Table 'fireflydb.password_resets' doesn't exist

### DISCUSSION THREAD

**Discussion by CappiSteijns** [firefly-iii/firefly-iii#6431] [Firefly III]
Hello,

I made a big whoopsie when trying to reset my password.

I'm running firefly on a docker container using Unraid. I forgot my password and do not have an email setup. 
I tried following ""Password reset issue #1611" where I had to Delete the entry from password_resets. But now I'm pretty sure I wasn't supposed to delete the whole table (or view, i'm not familiair with databases).

So now I keep getting this error:
![Firefly error](https://user-images.githubusercontent.com/84481241/189585868-735aa421-776d-4de2-ace4-2cc3f85e78d9.PNG)

These are the logs:
[docker logs password reset.txt](https://github.com/firefly-iii/firefly-iii/files/9545101/docker.logs.password.reset.txt)

I'm using adminer to edit the database, but I have no idea how to get the password_resets table back. 

Hope I can get some help.

**Comment by JC5** [comment]
You could set up a new instance and copy the "CREATE" statement from the `password_resets` table and apply it to your own database.
